data/utils.js

```diff
@@ -1,65 +1,138 @@
-function normalizeName(name) {
-  name = name.toLowerCase();
+import {
+  round, groupBy, sumBy, sortBy, orderBy, filter, first, last, isEmpty,
+  zipObject,
+} from 'lodash';
```

Comment:
If this doesn't fit on one line, each item should be on a separate line.

Reply:
Gotcha. Will change. Was surprised that the linter didn't force it/correct it, so assumed this was your pref.

you may want to configure this setting in the project's/your eslint config: https://eslint.org/docs/rules/object-curly-newline

Reply:
That rule actually doesn't work as desired here: https://stackoverflow.com/questions/58656094/eslint-rule-to-put-a-new-line-inside-import

There was a proposal to address that, but it was never merged: https://github.com/eslint/eslint/issues/9259#issuecomment-525289402